tests/functional/auth/scope/test_show_scope.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+import json
+import textwrap
+
+import pytest
+from globus_sdk._testing import load_response_set
+
+
+@pytest.mark.parametrize(
+    "scope_id_or_string",
+    (
+        "24f3dcbe-7655-4721-bc64-d1c5d635b9a1",
+        "https://auth.globus.org/scopes/actions.globus.org/hello_world",
+    ),
+)
+def test_show_scope(run_line, scope_id_or_string):
+    load_response_set("cli.scopes")
+
+    result = run_line(f"globus auth scope show {scope_id_or_string}")
```

Comment:
I notice that we have pytest params matched to the data here. I think it's fine but if you wanted to break the dependency up a bit, you can
```python
@pytest.mark.parametrize("lookup_method", ("id", "scope_string"))
def test_...:
    rset = load_response_set("cli.scopes")
    if lookup_method == "id":
        scope_arg = rset.metadata["hello_world_id"]
    elif lookup_method == "scope_string":
        scope_arg = rset.metadata["hello_world_string"]
    else:
        raise NotImplementedError(lookup_method)
```

It's totally optional, but if you'd like to do so, there's a path to doing this and potentially something similar to the expected output formulation.

Reply:
It doesn't look like a conditional is required.

```python
@pytest.mark.parametrize("metadata_key", ("hello_world_id", "hello_world_string"))
def test_...:
    rset = load_response_set("cli.scopes")
    scope_arg = rset.metadata[metadata_key]
```